spec/00-abstract.md

```diff
@@ -0,0 +1,6 @@
+## Abstract
+
+This specification defines a standard format to propagate context
+information for distributed traces. The specification standardizes how context
```

Comment:
```suggestion
information that enables distributed tracing scenarios. The specification standardizes how context
```

Reply:
Don't know how this helps to add clarity.

Reply:
this information enables distributed tracing scenarios, but also can be used for other purposes. That was my main point here

Reply:
I moved comment to the latest version so it will not be lost and easier for you to accept/reject it